src/api/database/models/map/attributesAndIncludes.ts

```diff
@@ -1,5 +1,6 @@
 export const landmarkAssessmentAttributes = [
     "score", 
     "markdown", 
-    "createdAt"
+    "createdAt",
+    "assessorId",
```

Comment:
no need to have assessorId in the attribute list. you can always retrieve the id using assessment.assessor.id